test/server/index.test.js

```diff
@@ -0,0 +1,190 @@
+const assert = require('assert');
+const supertest = require('supertest');
+const server = supertest.agent('http://localhost:8080');
+
+describe('missingEmail error test', function() {
+  it('Signup success', function(done) {
+    server
+      .post('/api/signup')
+      .send({'username': 'aliceShen12345',
+        'email': '',
+        'phonenumber': '',
+        'fullname': '',
+        'password': 'qqwweerr',
+      })
+      .expect('Content-type,/json/')
```

Comment:
i dont really understand this string. It doesnt mean that it is wrong. Can you explain to me what does this do?